packages/web-components/fast-components/src/anchor/index.ts

```diff
@@ -50,7 +50,7 @@ export class Anchor extends FoundationAnchor {
      * @internal
      *
      */
-    public defaultSlottedContentChanged(oldValue, newValue): void {
+    public defaultSlottedContentChanged(oldValue: unknown, newValue: SVGElement): void {
```

Comment:
The slot content is _often_ an SVG, but this should not be required. The slots should allow any content, including plain text or any other component.

Also, does the `unknown / something` convention assume that it's always null or undefined and then it's _something_? What if the content is unset?

Reply:
Unknown is actually unknown :) - not a great explanation, I know. It’s actually semantic in Typescripts case. In terms of the new value, agree that this could be anything. In the method we are filtering to apply logic when it’s only an svg, but it could be any HTML Element or a text node..